crates/vsock/README.md

```diff
@@ -44,7 +44,6 @@ vhost-user-vsock --guest-cid=<CID assigned to the guest> \
   --socket=<path to the Unix socket to be created to communicate with the VMM via the vhost-user protocol>
   --uds-path=<path to the Unix socket to communicate with the guest via the virtio-vsock device>
   [--tx-buffer-size=<size of the buffer used for the TX virtqueue (guest->host packets)>]
-  --config=<path to the local yaml configuration file>
```

Comment:
Here with the rebase you lost the description of `--vm` and removed `--config` by accident.